workspace/commands/helpers.py

```diff
@@ -60,17 +60,23 @@ def envsection(self, env=None):
 
     @property
     def workdir(self):
-        return os.path.join(self.path, self.get('tox', 'toxworkdir', '.tox'))
+        toxworkdir = self.get('tox', 'toxworkdir', '.tox')
+        toxworkdir = toxworkdir.replace('{homedir}', self.homedir)
+        toxworkdir = toxworkdir.replace('{toxinidir}', self.path)
+        return os.path.join(self.path, toxworkdir)
```

Comment:
`self.expand_vars` should take care of expanding `homedir`, can you switch to use that? As for toxinidir, let's add that as a property, then expand_vars can just work as well.